sdk/storage/src/adls_gen2/filesystem/requests/delete_filesystem_builder.rs

```diff
@@ -176,34 +98,28 @@ where
     }
 }
 
-impl<'a, C, FilesystemSet> DeleteFilesystemBuilder<'a, C, FilesystemSet>
-where
-    FilesystemSet: ToAssign,
-    C: Client,
-{
-}
-
 impl<'a, C> DeleteFilesystemBuilder<'a, C, Yes>
 where
     C: Client,
 {
     pub async fn finalize(self) -> Result<DeleteFilesystemResponse, AzureError> {
```

Comment:
Why are these still called `finalize`?

Reply:
I haven't touched the adls_gen2 code (aside for obvious changes to make it compile). We will address them in a separate PR (this one is already big as it is).